docs/quick_start.md

```diff
@@ -51,69 +83,46 @@ git commit -m "Commit message goes here!"
 
 *Checkout the [Version Control](https://brown-ccv.github.io/honeycomb-docs/docs/version_control) page for more information about working with git*
 
-### 3. Install Electron dependencies
-
-Honeycomb relies on Electron to package the cross-platform desktop applications. Before starting, you will need to install Electron's pre-requisites below. You can find the full instructions on the [electron documentation](https://www.electronjs.org/docs/development/build-instructions-gn) for your specific OS.
 
-#### Windows
 
-- Visual Studio:
-Install the latest version of [Visual Studio](https://visualstudio.microsoft.com/downloads/) with the Desktop Development for C++ Workflow. 
-To add the workflow, follow [these instructions](https://docs.microsoft.com/en-us/cpp/build/vscpp-step-0-installation?view=msvc-160#:~:text=If%20you%20have%20Visual%20Studio,Then%2C%20choose%20Modify).
+### 3. Install NPM Packages
+With Node.js installed in Step 0, we are now able to utilize the `npm` command in the terminal.
+`npm` is a command-line utility for Node.js that makes it easy for you to install packages and maintain them throughout the lifecycle of your application. 
 
-- Node.js:
-Install [Node.js](https://nodejs.org/en/download/)
+Please run the following command to trigger the installation of all the dependencies relevant to this application.
 
-- Git bash (Git 2.20.0 or later with support for "--show-current"):
-Install [git bash](https://git-scm.com/downloads)
-
-**Note: restart computer after all installs are complete**
-
-#### MacOS
+```
+npm install
+```
 
-- Command Line Tools: Type in the terminal
+Honeycomb, like most applications, contains a template called `package.json`. This file contains three distinctive parts that we interact with using a series of npm commands.
 
-    ```
-    xcode-select --install
-    ```
 
-- Node.js:
-Install [Node.js](https://nodejs.org/en/download/)
+##### 1. Metadata
+![](assets/packagelock1.png)
```

Comment:
I think we might be best putting in the pure code here, rather than a screenshot.

```json
{
  "name": "honeycomb",
  "description": "all-in-one task starter app with jsPsych + React + Electron + psiturk ",
  "author": {
    "name": "Brown CCV",
    "email": "ccv-bot@brown.edu",
    "url": "ccv.brown.edu"
  },
```